core/src/main/java/com/sequenceiq/cloudbreak/converter/spi/InstanceMetaDataToCloudInstanceConverter.java

```diff
@@ -77,11 +77,11 @@ public CloudInstance convert(InstanceMetaData metaDataEntity, DetailedEnvironmen
 
     private InstanceStatus getInstanceStatus(InstanceMetaData metaData) {
         switch (metaData.getInstanceStatus()) {
-
             case REQUESTED:
                 return InstanceStatus.CREATE_REQUESTED;
             case CREATED:
                 return InstanceStatus.CREATED;
+            case STARTED:
```

Comment:
First impression: I think it is better to have a new STARTED state, instead of re-using SERVICES_UNHEALTHY (maybe even SERVICES_STARTING).
1) Less confusing to the users when performing these operations.
2) Easier for PERISCOPE to query this state. (That said, Periscope may need to handle many of the SERVICE_* states anyway)